packages/integration-sdk-dev-tools/README.md

```diff
@@ -0,0 +1,71 @@
+# @jupiterone/integration-sdk-dev-tools
+
+This package contains some common dependencies and configuration files used in
+JupiterOne integrations.
+
+This module is opinionated and bundles in the following dependencies:
+
+- `typescript`
+- `prettier`
+- `jest`
+- `eslint`
+- `husky`
+- `lint-staged`
+
+For convenience, this package also comes bundled with the
+`@jupiterone/integration-sdk-cli` module.
```

Comment:
I wonder if this is the best thing to do for folks who will be using the CLI without a need for all these dev tools, but perhaps that isn't ever going to be the case.

Reply:
The `integration-sdk-cli` package will be more useful if we decide to tack on the `integration` subcommand to the next gen version of the `j1` cli. Idk if doing that will be very useful, but it could be depending on what functionality we decide to add to the `integration-sdk-cli` (external integration stuff). 

For now though, it's just kinda nice having really small, well focused packages.

Reply:
I may be misunderstanding your comment... I totally love the idea of a `-cli` package, and small packages.

I was only wondering if the `-cli` needs to always come with the `-dev-tools`, which appear to be all about formatting/managing code in an integration project. I suppose there is no use of the `-cli` today without maintaining some code, so it is fine for them to travel together in that way. Of course, the `-cli` itself will have the `-dev-tools` as it's own `devDependencies` as well, I just think it may be more clear to have the integration projects depend on both themselves in `devDependencies` (not depend on the transitive property).

Reply:
ah okay sorry I think I misunderstood your original comment.

So I originally thought about making `dev-tools` a project that only contained common config and development deps, but then @philidem made a comment about bundling the cli with it (in integrations slack channel). After thinking about it a little, it kinda made sense to just include it in the `dev-tools` package because the CLI is really is just a kind of development tool. Our own internal `dev-tools` package comes with a few cli commands for doing various dev related things so it seemed to fit the pattern.

I don't feel too strongly about having the cli bundled in the `dev-tools` project though so if you would prefer we keep the `dev-tools` project a little more pure, I'm down remove the CLI from it.

Reply:
Oh, haha, I misunderstood, thinking the `-cli` came with the `-dev-tools`, instead of what you've done (`-dev-tools` comes with `-cli`), which is excellent. Sorry!